developer/src/kmc/src/util/NodeCompilerCallbacks.ts

```diff
@@ -121,6 +122,14 @@ export class NodeCompilerCallbacks implements CompilerCallbacks {
 
     this.messages.push({...event});
 
+    // report fatal errors to Sentry, but don't display; note, it won't be
+    // reported if user has disabled the Sentry setting
```

Comment:
I think it's saying don't display *here* but it will display once the reporting resolves.  Or it will display once the caller sees it and unpacks the lists of messages.

It may be a little misleading to say 'don't display' because it sounds like fatal errors are never displayed .which would be surprising if they were reported but not displayed!